### DISCUSSION TITLE
🚀 Feature Request: Add a delay option for showing/hiding the Mega Menu content (Related to #23430)

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21353] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Currently, I use the jet menu plugin to create mega menu. But there is a simple yet important problem.

some customers complaining over that the Mega Menu opens every time mouse passes over menu. This is really annoying for desktop users.

### Describe the solution you'd like

 it would be great if you add an option to set the mega menu display delay. For example, using this option we can specify that when the mouse is placed on the menu, the mega menu will appear with a delay of 100ms.
This feature prevents an annoying incident for users.

thanks

#### [Edit by @nicholaszein]:

Some users also have difficulty hovering the content of a menu item if they place the mouse slightly outside the hover area. Adding an option to also set a delay for closing the menu item content would prevent issues when trying to access it when the mouse passes outside the menu item or the content, closing it prematurely.

**Comment by nicholaszein** [comment]
This request was mentioned here: 

- #23430
- #23946

**Comment by anoxarc** [comment]
Would be great to have this functionality - UPVOTE
